pkg/wrangler/context.go

```diff
@@ -104,12 +105,27 @@ var (
 	}
 	AddToScheme = localSchemeBuilder.AddToScheme
 	Scheme      = runtime.NewScheme()
+
+	cacheSyncTimeoutEnvVar = "CACHE_SYNC_TIMEOUT"
+	cacheSyncTimeout       time.Duration
```

Comment:
You could assign `time.Minute * 5` here and then in the init, simplify the code to  check for the env-var and assign if it's set (and valid)?